runelite-client/src/test/java/net/runelite/client/plugins/woodcutting/WoodcuttingPluginTest.java

```diff
@@ -127,4 +133,96 @@ public void testBirdsNest()
+		Tile tile = mock(Tile.class);
+		TileItem eliteTileItem = mock(TileItem.class);
+		when(eliteTileItem.getId()).thenReturn(ItemID.CLUE_NEST_ELITE);
+		ItemSpawned eliteClueSpawned = new ItemSpawned(tile, eliteTileItem);
+
+		when(woodcuttingConfig.showNestNotification()).thenReturn(true);
+		when(woodcuttingConfig.clueNestNotifyTier()).thenReturn(ClueNestTier.BEGINNER);
+		woodcuttingPlugin.onItemSpawned(eliteClueSpawned);
+		woodcuttingPlugin.onChatMessage(nestChatMessage);
+		woodcuttingPlugin.onGameTick(null);
+		verify(notifier).notify("A bird nest has spawned!");
+	}
+
+	@Test
+	public void testClueNestDisabledConfig()
+	{
+		ChatMessage nestChatMessage = new ChatMessage(null, ChatMessageType.GAMEMESSAGE, "", BIRDS_NEST_MESSAGE, "", 0);
+		Tile tile = mock(Tile.class);
+		TileItem eliteTileItem = mock(TileItem.class);
+		when(eliteTileItem.getId()).thenReturn(ItemID.CLUE_NEST_ELITE);
+		ItemSpawned eliteClueSpawned = new ItemSpawned(tile, eliteTileItem);
+
+		when(woodcuttingConfig.showNestNotification()).thenReturn(true);
+		when(woodcuttingConfig.clueNestNotifyTier()).thenReturn(ClueNestTier.DISABLED);
+		woodcuttingPlugin.onItemSpawned(eliteClueSpawned);
+		woodcuttingPlugin.onChatMessage(nestChatMessage);
+		woodcuttingPlugin.onGameTick(null);
+		verifyNoMoreInteractions(notifier);
+	}
+
+	@Test
+	public void testClueNestConfigLargerThanSpawn()
+	{
+		ChatMessage nestChatMessage = new ChatMessage(null, ChatMessageType.GAMEMESSAGE, "", BIRDS_NEST_MESSAGE, "", 0);
+		Tile tile = mock(Tile.class);
+		TileItem beginnerTileItem = mock(TileItem.class);
+		when(beginnerTileItem.getId()).thenReturn(ItemID.CLUE_NEST_BEGINNER);
+		ItemSpawned beginnerClueSpawned = new ItemSpawned(tile, beginnerTileItem);
+
+		when(woodcuttingConfig.showNestNotification()).thenReturn(true);
+		when(woodcuttingConfig.clueNestNotifyTier()).thenReturn(ClueNestTier.HARD);
+		woodcuttingPlugin.onItemSpawned(beginnerClueSpawned);
+		woodcuttingPlugin.onChatMessage(nestChatMessage);
+		woodcuttingPlugin.onGameTick(null);
+		verifyNoMoreInteractions(notifier);
+	}
+
+	@Test
+	public void testClueNestPlayerDrop()
+	{
+		ChatMessage nestChatMessage = new ChatMessage(null, ChatMessageType.GAMEMESSAGE, "", BIRDS_NEST_MESSAGE, "", 0);
+		Tile tile = mock(Tile.class);
+		TileItem beginnerTileItem = mock(TileItem.class);
+		when(beginnerTileItem.getId()).thenReturn(ItemID.CLUE_NEST_BEGINNER);
+		ItemSpawned beginnerClueSpawned = new ItemSpawned(tile, beginnerTileItem);
+		TileItem nestTileItem = mock(TileItem.class);
+		when(nestTileItem.getId()).thenReturn(ItemID.BIRD_NEST_22798);
+		ItemSpawned regularNestSpawned = new ItemSpawned(tile, nestTileItem);
+
+		when(woodcuttingConfig.showNestNotification()).thenReturn(true);
+
+		// Player drops nest
+		woodcuttingPlugin.onItemSpawned(beginnerClueSpawned);
+		// Assume that at least one tick has passed
+		woodcuttingPlugin.onGameTick(null);
```

Comment:
This should verify no interactions have happened with `notifier` at this point.
```suggestion
		// Player drops nest
		woodcuttingPlugin.onItemSpawned(beginnerClueSpawned);
		woodcuttingPlugin.onGameTick(null);
		verifyNoInteractions(notifier);

```